src/Native/Native.proj

```diff
@@ -143,8 +143,10 @@
   </Target>
 
   <Target Name="Pack" />
-  <Target Name="Restore" />
   <Target Name="Test" />
 
+  <ItemGroup>
+    <PackageDownload Include="MlNetMklDeps" Version="[$(MlNetMklDepsVersion)]" />
```

Comment:
>[ [](http://example.com/codeflow?start=53&length=1)

interesting, does the version included inside brackets?

Reply:
Yes, PackageDownload requires explicit version references. https://learn.microsoft.com/en-us/nuget/consume-packages/packagedownload-functionality